mmv1/products/discoveryengine/DataStore.yaml

```diff
@@ -0,0 +1,145 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'DataStore'
+description: |
+  Vertex AI Search and Conversation data store is a collection of websites and
+  documents used to find answers for end-user's questions.
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    'Create a search data store': 'https://cloud.google.com/generative-ai-app-builder/docs/create-data-store-es'
+  # TODO: Update the link to the REST API reference for the resource from v1alpha to v1.
+  api: 'https://cloud.google.com/generative-ai-app-builder/docs/reference/rest/v1alpha/projects.locations.collections.dataStores'
+
+base_url: 'projects/{{project}}/locations/{{location}}/collections/default_collection/dataStores'
```

Comment:
Hi @jialei-chen , I think the `base_url` should be 
```
'projects/{{project}}/locations/{{location}}/collections/default_collection/dataStores/{{data_store_id}}'
```

Reply:
Hi @5fff, thanks. As for this error
```
vcr_utils.go:152: Failed state verification, resource with ID projects/codelab-terraform-jialei/locations/global/collections/default_collection/dataStores/ not found
```
I fixed it by adding import formats explicitly to the resource with the `{{data_store_id}}` variable"
```
import_format:
  [
     'projects/{{project}}/locations/{{location}}/collections/default_collection/dataStores/{{data_store_id}}',
  ]
```